### PR TITLE
BAEL-4212 (String) or .toString()?

### PR DESCRIPTION
**Pull request by Saber-k**

### CODE REVIEW

File: core-java-modules/core-java-string-operations-3/src/main/java/com/baeldung/tostring/StringCastUtils.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.baeldung.tostring;
+
+public class StringCastUtils {
+    public static String castToString(Object object) {
+        if (object instanceof String)
```
Comment (JonCook) [inline core-java-modules/core-java-string-operations-3/src/main/java/com/baeldung/tostring/StringCastUtils.java:5]: Let's make sure we include parenthesis for the if statements. Generally we follow that rule for our samples.

File: core-java-modules/core-java-string-operations-3/src/test/java/com/baeldung/tostring/ToStringUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
+package com.baeldung.tostring;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (JonCook) [inline core-java-modules/core-java-string-operations-3/src/test/java/com/baeldung/tostring/ToStringUnitTest.java:5]: Let's avoid * imports and import only the things we need.

File: core-java-modules/core-java-string-operations-3/src/main/java/com/baeldung/tostring/StringCastUtils.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+            return (String) object;
+        return null;
+    }
+
+    public static String getToString(Object object) {
```
Comment (JonCook) [inline core-java-modules/core-java-string-operations-3/src/main/java/com/baeldung/tostring/StringCastUtils.java:10]: Can we think of a better name for this method? getToString sounds a bit weird to me. What about something like objectToString or something a bit more meaningful? Open to ideas.

**Review by JonCook — CHANGES_REQUESTED** [review]
Hey @Saber-k - I left a few comments for review. Thanks.

File: core-java-modules/core-java-string-operations-3/src/main/java/com/baeldung/tostring/StringCastUtils.java (hunk 3 above)
Comment (Saber-k) [reply core-java-modules/core-java-string-operations-3/src/main/java/com/baeldung/tostring/StringCastUtils.java:10]: what about - getStringRepresentation

**Pull request merged**